docs/conf.py

```diff
@@ -14,20 +14,20 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
-import brickschema
+
+sys.path.insert(0, os.path.abspath(".."))
+import brickschema  # noqa: E402
 
 # -- Project information -----------------------------------------------------
 
-project = 'brickschema'
-copyright = '2020, Gabe Fierro'
-author = 'Gabe Fierro'
+project = "brickschema"
+copyright = "2021, Gabe Fierro"
```

Comment:
We should probably keep 2020 and 2021 in the copyright. You can also add yourself now, too!

Reply:
> You can also add yourself now, too!

:) 

I've updated the config. You may want to update/remove my name from any part that doesn't make sense.